Assets/Plugins/Source/Editor/Utility/PackageFileUtility.cs

```diff
@@ -208,20 +244,22 @@ private static IEnumerable<FileInfoMatchingResult> FindMatchingFiles(string root
             CancellationToken cancellationToken = default,
             Action<string> postProcessCallback = null)
         {
-            // TODO: A major improvement to this system would be to automatically include .meta files, and gracefully
-            //       handle instances where the meta file *should* exist, but does not - gracefully in this context
-            //       would mean that if the .meta file exists, it should just be copied over; whereas if there is a
-            //       file entry copied over that does not have a corresponding meta file in the source, it indicates
-            //       that a meta file entry is MISSING from the source, and that should stop the creation of a package.
```

Comment:
This `TODO` remark is accomplished by this PR, so it is another way to understand it if you would like. The only change from the implementation suggested above is a missing meta file does not result in a failed build but a logged warning.